### PR TITLE
[Asgardeo] Integrate JFrog - Add Jfrog scanning to Azure Devops pipeline

### PR DESCRIPTION
**Pull request by IsuruGunarathne**
image

## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: .azure/asgardeo-java-oidc-sdk-sca-scan.yaml [hunk 1]
```diff
@@ -45,3 +45,10 @@ jobs:
         parameters:
           API_KEY: $(FOSSA-API-KEY)
           BRANCH: master
+      - template: ci-pipelines/templates/sca-scan-jfrog.yaml@templates
+        parameters:
+          PROJECT_TYPE: mvn
+          GITHUB_CONNECTION: $(GITHUB_CONNECTION) # GitHub connection name to Show the vulnerability report as a PR comment
```
Comment (VimukthiPerera) [inline .azure/asgardeo-java-oidc-sdk-sca-scan.yaml:51]: This has to be GITHUB_CONNECTION: asgardeo-github-sca-scan

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: .azure/asgardeo-java-oidc-sdk-sca-scan.yaml [hunk 2]
```diff
@@ -45,3 +45,10 @@ jobs:
...
+          ACCESS_TOKEN: $(ACCESS_TOKEN) # JFrog access token
+          SERVER_ID: $(CONNECTION_NAME) # JFrog SERVER ID
```
Comment (VimukthiPerera) [inline .azure/asgardeo-java-oidc-sdk-sca-scan.yaml:53]: Move this and the following parameters to the template.

File: .azure/asgardeo-java-oidc-sdk-sca-scan.yaml (hunk 1 above)
Comment (IsuruGunarathne) [reply .azure/asgardeo-java-oidc-sdk-sca-scan.yaml]: Changed in commit 8b4a23d611b53927fcd8025dc07582f343997bd3

File: .azure/asgardeo-java-oidc-sdk-sca-scan.yaml (hunk 2 above)
Comment (IsuruGunarathne) [reply .azure/asgardeo-java-oidc-sdk-sca-scan.yaml]: Done, only Access token has to be provided as a parameter

**Review by VimukthiPerera — APPROVED** [review]

**Review by VimukthiPerera — APPROVED** [review]

**Review by Migara-Pramod — APPROVED** [review]

**Pull request merged**